src/browser/BrowserService.cpp

```diff
@@ -587,11 +663,9 @@ int BrowserService::sortPriority(const Entry* entry, const QString& host, const
 
 bool BrowserService::matchUrlScheme(const QString& url)
 {
-    QString str = url.left(8).toLower();
-    return str.startsWith("http://") ||
-           str.startsWith("https://") ||
-           str.startsWith("ftp://") ||
-           str.startsWith("ftps://");
+    QRegularExpression re("^(http|https|ftp|ftps)://");
```

Comment:
I was more thinking about a *truly* generic regular expression. Something along the lines of `"^[a-zA-Z][a-zA-Z0-9\\-\\+\\.]*://"`

Reply:
Understood :)